jaxrs/src/main/resources/kafka/producer.props

```diff
@@ -0,0 +1,9 @@
+bootstrap.servers=172.19.0.3:9092
```

Comment:
Agreed, just a quick fix for now but we will definitely have to address it in `AGPUSH-2104`